pelican/utils.py

```diff
@@ -546,12 +543,12 @@ def file_times(path):
             dirs[:] = [x for x in dirs if not x.startswith(os.curdir)]
 
             for f in files:
-                if (f.endswith(tuple(extensions)) and
-                    not any(fnmatch.fnmatch(f, ignore) for ignore in ignores)):
-                    try:
-                        yield os.stat(os.path.join(root, f)).st_mtime
-                    except OSError as e:
-                        logger.warning('Caught Exception: %s', e)
+                if f.endswith(tuple(extensions)) and \
+                   not any(fnmatch.fnmatch(f, ignore) for ignore in ignores):
+                        try:
```

Comment:
the extra indentation is added to distinguish from the not line before; else there would only be 1 space or so difference, which makes it hard to read